org.yaml

```
@@ -115,6 +115,30 @@ orgs:
         privacy: closed
         repos:
           kgateway.dev: maintain
+      kgateway-api-owners:
+        description: Maintainers of the kgateway envoy API
+        maintainers:
+        - howardjohn
+        - jenshu
+        - lgadban
+        - shashankram
+        - timflannagan
+        - yuval-k
```

hrm, i'm not sure if peribolos will accept this. we might need to split it up like the other groups according to who is an org admin:
```suggestion
        maintainers:
        - jenshu
        - lgadban
        - timflannagan
        - yuval-k
        members:
        - howardjohn
        - shashankram
```

---

I think it will based on the example in their readme? https://github.com/uwu-tools/peribolos?tab=readme-ov-file 
```
teams:
  application-admins:
    description: admin access to application
    maintainers:
    - kow3ns
    members:
    - mattfarina
    - prydonius
    privacy: closed
  architecture-tracking-admins:
    description: admin permission for architecture-tracking
    maintainers:
    - jdumars
    - bgrant0607
    privacy: closed
  # etc
```

I think having a distinct member/maintainer grouping for this is extra overhead that doesn't make sense.